packages/core/lib/core.ts

```diff
@@ -68,6 +68,16 @@ export function createClient<S extends CoreSchema, C extends Configuration, T> (
     start: (config: C | string) => {
       const configuration = validateConfig<S, C>(config, options.schema)
 
+      if (configuration.bugsnag) {
```

Comment:
Is it worth checking here that `setTraceCorrelation` exists so we avoid even adding the callback if the bugsnag-js version is too old?

Reply:
oooh, that's even better!

Reply:
although, it appears I'm not entirely sure how to check for that...

Reply:
I think you can check for `configuration.bugsnag.Event.prototype.setTraceCorrelation`:

https://github.com/bugsnag/bugsnag-js/blob/76f50d9b0684d8a445829c932fc182ee191775ab/packages/browser/src/notifier.js#L105

Reply:
Do you think that would be best placed here, or in the config validation?

Reply:
Sorry, missed this!

I think here is best — if/when we add more stuff that needs APIs from `Bugsnag`, we probably don't want to disable _all_ the error integrations if new APIs are missing

Reply:
Raised PLAT-12350 to cover this, so we can get this ticket merged down